src/RapideFix/Validation/ValidatorCollection.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using RapideFix.DataTypes;
+
+namespace RapideFix.Validation
+{
+  public class ValidatorCollection : IValidator
+  {
+    private readonly List<IValidator> _validators;
+
+    public ValidatorCollection(IntegerToFixConverter integerConverter)
+    {
+      _validators = new List<IValidator>();
+      _validators.Add(new FixVersionValidator());
```

Comment:
I think we should somehow make it more obvious, that the order of the validators matters, as for example, `LengthValidator` depends on the information set on `FixMessageContext` by previous validators. I Just have the feeling, that this not expected looking at this class.

Reply:
I am moving the creating of FixMessageContext ahead of validation. I do not want to have any dependency between the validators if possible.